sdk/metricsadvisor/azure-ai-metricsadvisor/README.md

```diff
@@ -10,6 +10,8 @@ Metrics Advisor is a scalable real-time time series monitoring, alerting, and ro
 
 ## _Disclaimer_
 
+_Python 3.7 or later is required to use this package. For more details, please refer to [Azure SDK for Python version support policy](https://github.com/Azure/azure-sdk-for-python/wiki/Azure-SDKs-Python-version-support-policy)._
```

Comment:
Like with other packages, I think we can omit his from the README. Dropping 2.7 was a special case.